docs/pages/management/guides/awsoidc-integration.mdx

```diff
@@ -0,0 +1,143 @@
+
+Those files should be exposed in S3, for which we'll need to pick a bucket <Var name="s3-bucket"/> and a prefix <Var name="s3-prefix"/>.
+
+Download the current configuration and public keys:
+```code
+$ mkdir .well-known
+$ curl https://<Var name="teleport.example.com"/>/.well-known/openid-configuration > .well-known/openid-configuration
+$ curl https://<Var name="teleport.example.com"/>/.well-known/jwks-oidc > .well-known/jwks
+```
+
+Edit the `openid-configuration` and replace the `issuer` and `jwks_uri` fields with with the S3 location:
+```json
+{
+  "issuer": "<Var name="s3-bucket"/>.s3.amazonaws.com/<Var name="s3-prefix"/>",
+  "jwks_uri": "https://<Var name="s3-bucket"/>.s3.amazonaws.com/<Var name="s3-prefix"/>/.well-known/jwks",
+  // other fields
+}
+```
+
+Upload those files to `<Var name="s3-bucket"/>` bucket:
+```code
+<Var name="s3-bucket"/>
+└── <Var name="s3-prefix"/>
+    └── .well-known
+        ├── openid-configuration
+        └── jwks
+```
+
+Make those objects public (either using Bucket Policies or ACLs):
+```bash
+$ curl https://<Var name="s3-bucket"/>.s3.amazonaws.com/<Var name="s3-prefix"/>/.well-known/openid-configuration
+$ curl https://<Var name="s3-bucket"/>.s3.amazonaws.com/<Var name="s3-prefix"/>/.well-known/jwks
+```
+
+## Step 3/5. Configure the Identity Provider in AWS
+Navigate to [AWS IAM Identity Provider](https://console.aws.amazon.com/iam/home#/identity_providers/create) and configure the Identity Provider:
+- Provider type: OpenID Connect
+- Provider URL:
+```code
+https://<Var name="s3-bucket"/>.s3.amazonaws.com/<Var name="s3-prefix"/>
+```
+- Audience: `discover.teleport`
+
+## Step 4/5. Create IAM Role
+An IAM Role must be created to assign the required policies to the Integration <Var name="iam-role"/>.
+
+This IAM Role is created without any policy, as those are added depending on the feature you would like to use.
+However, it must be configured to allow the Identity Provider to assume it.
+To achieve this, add the following Trust Relationship:
+```json
+{
+	"Version": "2012-10-17",
+	"Statement": [
+		{
+			"Effect": "Allow",
+			"Principal": {
+				"Federated": "arn:aws:iam::<Var name="aws-account-id" description="AWS Account ID"/>:oidc-provider/<Var name="s3-bucket"/>.s3.amazonaws.com/<Var name="s3-prefix"/>"
+			},
+			"Action": "sts:AssumeRoleWithWebIdentity",
+			"Condition": {
+				"StringEquals": {
+					"<Var name="s3-bucket"/>.s3.amazonaws.com/<Var name="s3-prefix"/>:aud": "discover.teleport"
+				}
+			}
+		}
+	]
+}
+```
+
+It is also required to add the following tags, which are used by Teleport to ensure it can change the IAM Role when onboarding new features:
+```code
+teleport.dev/cluster      <Var name="cluster-name"/>
+teleport.dev/origin       integration_awsoidc
+teleport.dev/integration  <Var name="my-integration"/>
+```
+
+## Step 5/5. Create Integration resource
+```yaml
```

Comment:
"Create integration resource" and just giving them yaml will confuse users. We should show them how to create it with tctl.